clients/stacks/stacks.go

```diff
@@ -0,0 +1,265 @@
+package stacks
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+	"github.com/influxdata/influx-cli/v2/pkg/template"
+)
+
+type Client struct {
+	clients.CLI
+	api.StacksApi
+	api.OrganizationsApi
+	api.TemplatesApi
+}
+
+type ListParams struct {
+	OrgId   string
+	OrgName string
+
+	StackIds   []string
+	StackNames []string
+}
+
+func (c Client) List(ctx context.Context, params *ListParams) error {
+	if params.OrgId == "" && params.OrgName == "" && c.ActiveConfig.Org == "" {
+		return clients.ErrMustSpecifyOrg
+	}
+
+	orgId := params.OrgId
```

Comment:
There is some copy-paste here (and across all the `clients` code). I intend to clean it up across the board after all the commands have been ported over, as part of #132